src/site/content/en/fast/efficiently-load-third-party-javascript/index.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: post
+title: Efficiently load third-party JavaScript
+subhead: Avoid common pitfalls of using third-party scripts and improve load times and user experience.
+authors:
+  - mihajlija
+date: 2019-07-29
+description: |
+  Learn how to avoid common pitfalls of using third-party scripts and improve load times and user experience.
+hero: hero.jpg
+alt: Aerial view of shipping containers.  
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - performance
+  - third-party
+  - javascript
+---
+
+If a third-party script is [slowing down](/third-party-javascript/) your page load, you have two options to improve performance:
+
+* Remove it if it doesn't add clear value to your site.
+
+* Optimize the loading process.
+
+This guide explains how to optimize the loading process of third-party scripts with the following techniques: 
```

Comment:
```suggestion
This post explains how to optimize the loading process of third-party scripts with the following techniques: 
```
web.dev goes with "post" rather than "guide." I'm adding it to our word list :)